src/main/java/com/sekwah/advancedportals/portals/Portal.java

```diff
@@ -8,7 +8,13 @@
 import com.sekwah.advancedportals.api.portaldata.PortalArg;
 import com.sekwah.advancedportals.destinations.Destination;
 import com.sekwah.advancedportals.effects.WarpEffects;
-import org.bukkit.*;
+import org.bukkit.Bukkit;
```

Comment:
There's not much point of importing each individually like this. This only effects the bytecode so there is no performance change at runtime.

Reply:
Actually Ill just re-optimise the imports before I publish the next version